nx/source/services/vi.c

```diff
@@ -1,11 +1,14 @@
+#define NX_SERVICE_ASSUME_NON_DOMAIN
 #include <string.h>
-#include "types.h"
-#include "result.h"
-#include "kernel/ipc.h"
-#include "runtime/hosversion.h"
+#include "service_guard.h"
 #include "services/applet.h"
 #include "services/vi.h"
 #include "display/parcel.h"
+#include "runtime/hosversion.h"
+
+typedef struct {
```

Comment:
I'm thinking this should be moved to vi.h instead, and used in the relevant structs.